api/app/models/spree/webhooks/event.rb

```diff
@@ -1,9 +1,10 @@
 module Spree
   module Webhooks
     class Event < Spree::Webhooks::Base
-      validates :name, :subscriber, presence: true
+      validates :name, presence: true
+      validates :subscriber, presence: true, on: :update
 
-      belongs_to :subscriber, inverse_of: :events, optional: false
+      belongs_to :subscriber, inverse_of: :events, optional: true
```

Comment:
@sebastian-palma why are we changing this to optional? we're aware of the subscriber before sending the event

Reply:
That's right, I just added at the "beginning" of the flow, I'll move it to HandleRequest so the insert goes in the background as well.